be/src/runtime/file_result_writer.h

```diff
@@ -58,6 +58,10 @@ struct ResultFileOptions {
     int write_buffer_size_kb;
     THdfsProperties hdfs_properties;
     bool use_broker;
+    int64_t max_row_group_bytes{0};
+    bool use_dictory{true};
```

Comment:
what about file size? Do we support rolling file

Reply:
max_file_size_bytes is 1GB in file_result_writer.h